oct/cli/package/aws.py

```diff
@@ -71,21 +80,22 @@ def ami(context, upgrade_stage, mark_ready, tags):
         else:
             key = tag
         ami_tags[key] = value
+    playbook_variables['origin_ci_aws_additional_tags'] = ami_tags
```

Comment:
Should be after the `if mark_ready:` block

Reply:
doesn't matter, `ami_tags` is stored by reference in the `playbook_variables` hash

Reply:
Oh is it? Whatever, Python